napalm_ros/ros.py

```diff
@@ -363,6 +364,19 @@ def get_facts(self):
             ),
         }
 
+    def get_config(self, retrieve='all', full=False, sanitized=False):
```

Comment:
Should the defaults here be True?

```suggestion
    def get_config(self, retrieve='all', full=True, sanitized=True):
```

Reply:
No.